test/functional/fixtures/lua/test_plug/health/lua_success.lua

```diff
@@ -0,0 +1,10 @@
+local M = {}
+
+M.check = function()
+  vim.fn['health#report_start']("report 1")
+  vim.fn['health#report_ok']("everything is fine")
+  vim.fn['health#report_start']("report 2")
+  vim.fn['health#report_ok']("nothing to see here")
```

Comment:
we might as well take this opportunity to introduce a `lua/health.lua` module with proper Lua `health.report_xx()` functions. They can just wrap `vim.fn['...']` for now, but let's not perpetuate that in the Lua healthchecks themselves.